spec/README.md

```diff
@@ -53,9 +53,12 @@ all authoring guidelines, diagrams, examples, and notes.
 
 The key words MAY, MUST, MUST NOT, OPTIONAL, RECOMMENDED,
 SHOULD, and SHOULD NOT in this document are to be interpreted as
-described in BCP 14 [RFC2119] [RFC8174].
+described in BCP 14 \[[RFC2119](https://www.ietf.org/rfc/rfc2119.txt)\] \[[RFC8174](https://www.ietf.org/rfc/rfc8174.txt)\].
```

Comment:
This paragraph should probably be replaced by the text recommended in RFC 8174, rather than being a slightly modified version of the text recommended in RFC 2119.